src/gameHandlers/getCorrectAnswerBasedOnLocalStorage.ts

```diff
@@ -0,0 +1,25 @@
+import { CountryFlags, CurrentVariants, SeenFlagsIdList } from "types";
+import {
+  getDataFromLocalStorage,
+  StorageKeys,
+} from "services/getDataFromLocalStorage";
+
+export const getCorrectAnswerBasedOnLocalStorage = (
+  countryFlags: CountryFlags
+) => {
+  const {
+    correct: correctVariantID,
+  } = getDataFromLocalStorage<CurrentVariants>(StorageKeys.CurrentVariants, {
+    correct: null,
+    answerVariants: [],
+  });
+  const seenFlags = getDataFromLocalStorage<SeenFlagsIdList>(
+    StorageKeys.SeenFlags,
+    []
+  );
+
+  if (!correctVariantID || seenFlags.includes(correctVariantID))
+    return undefined;
```

Comment:
do not need to return undefined, the same as we write only return;

Reply:
I would say it is more a matter of taste.
Some people prefer explicit over implicit returns. In my opinion, the explicit return is more clear and readable, so I would prefer to keep it explicit.